pkg/api/types/containerd_types.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
> `containerd_types.go`

`cri_types.go` might be more descriptive

Reply:
updated